program_transformers/src/mpl_core_program/mod.rs

```diff
@@ -30,6 +30,9 @@ pub async fn handle_mpl_core_account<'a, 'b, 'c>(
             )
             .await?
             {
+                if skip_metadata_json_download(&info, db).await {
+                    return Ok(());
+                }
                 download_metadata_notifier(info)
                     .await
                     .map_err(ProgramTransformerError::DownloadMetadataNotify)?;
```

Comment:
grpc-ingest and plerkle-ingest use different notifiers. If we can I'd like to apply this to grpc-ingest and ops. Idea here is to reduce the changes applied to plerkle with this update.